### ISSUE TITLE
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#867]
**Describe the bug**
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Dashboard screen.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash)
2.  Go to different dashboard gadgets such as "Process Monitor". You will see different buttons such as "Refresh" button.
3. When you mouse hover on them, you will see their title or label.
4. Now try move the pointer over the title or label, the label disappears.

**Expected behavior**
User should be able to move pointer over the additional content (tool tip, label etc.), and this additional content should not disappear, while moving focus on it.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
There is no additional content but title. So, closing this bug.
